app/code/core/Mage/Adminhtml/Block/Widget/Grid.php

```diff
@@ -220,6 +220,13 @@ class Mage_Adminhtml_Block_Widget_Grid extends Mage_Adminhtml_Block_Widget
         ],
     ];
 
+    /**
+     * Allow edit status
```

Comment:
```suggestion
     * This array caches the status of the isAllow() method, for every acl path
```

or something like that